src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -908,8 +908,8 @@ This version used in: {2}</value>
   <data name="Remove_separators" xml:space="preserve">
     <value>Remove separators</value>
   </data>
-  <data name="Generate_constructor" xml:space="preserve">
-    <value>Generate constructor...</value>
+  <data name="Generate_constructor_from_members" xml:space="preserve">
+    <value>Generate constructor from members...</value>
```

Comment:
Changed this string to be clearer about what selecting it will do.  Note '...' is our consistent suffix for indicating the user will get a dialog to do the work with.  In this case, the new title is indicating that invokign this will give them the dialog taht allows them to pick the members they want the constructor to initialize.